packages/shopify-app-session-storage-drizzle/src/adapters/drizzle-sqlite.adapter.ts

```diff
@@ -0,0 +1,137 @@
+import {Session} from '@shopify/shopify-api';
+import {SessionStorage} from '@shopify/shopify-app-session-storage';
+import {
+  InferInsertModel,
+  InferSelectModel,
+  desc,
+  eq,
+  inArray,
+} from 'drizzle-orm';
+import {BaseSQLiteDatabase} from 'drizzle-orm/sqlite-core';
+
+import {SQLiteSessionTable} from '../schemas/sqlite.schema';
+
+export class DrizzleSQLiteAdapter<
+  T extends BaseSQLiteDatabase<'sync' | 'async', TRunResult, TSchema>,
+  TRunResult,
+  TSchema extends {[key: string]: unknown} = {[key: string]: unknown},
+> implements SessionStorage
+{
+  constructor(
+    private readonly db: T,
+    private readonly sessionTable: SQLiteSessionTable,
```

Comment:
I assume this means the user can override the session table if they want to, right? That'd be a good thing to have.